packages/web-components/src/components/va-modal/va-modal.tsx

```diff
@@ -178,7 +178,7 @@ export class VaModal {
   /**
    * Save focusable children within the modal. Populated on setup
    */
-  @State() focusableChildren: HTMLElement[] = null;
+  focusableChildren: HTMLElement[] = null;
```

Comment:
I'm just curious why `@State()` needed to be removed here?

Reply:
It was causing an error/warning about the state updating during the wrong part of the lifecycle which could trigger unnecessary re-renders. All of the logic still works just fine without it being a 'state' variable, and it only ever gets updated once.